pymc3/tests/test_distributions.py

```diff
@@ -889,19 +889,25 @@ def test_half_flat(self):
         assert 0.0 == HalfFlat.dist().logcdf(np.inf).tag.test_value
         assert -np.inf == HalfFlat.dist().logcdf(-np.inf).tag.test_value
 
-    def test_normal(self):
+    def test_normal_logp(self):
         self.check_logp(
             Normal,
             R,
             {"mu": R, "sigma": Rplus},
             lambda value, mu, sigma: sp.norm.logpdf(value, mu, sigma),
             decimal=select_by_precision(float64=6, float32=1),
+            n_samples=-1,
```

Comment:
Passed as expected in https://github.com/pymc-devs/pymc3/pull/4497/checks?check_run_id=2021551266